pkg/domain/infra/abi/artifact.go

```diff
@@ -124,6 +125,10 @@ func (ir *ImageEngine) ArtifactRm(ctx context.Context, opts entities.ArtifactRem
 	for _, namesOrDigest := range namesOrDigests {
 		artifactDigest, err := artStore.Remove(ctx, namesOrDigest)
 		if err != nil {
+			if opts.Ignore {
+				logrus.Debugf("Artifact does not exist, ignoring error (--ignore): %v", err)
```

Comment:
Maybe include namesOrDigest in the error somewhere so it's very clear what artifact wasn't there